AWS: update win_gfx workstations to use base OS

Edwin-Pau opened the pull request:
Instead of using subscription-required Windows OS for Windows
graphics workstations, it now uses the same base OS as standard
Windows workstations and installs the required Nvidia graphics
drivers manually as part of the startup script.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
AWS: update win_gfx workstations to use base OS

Instead of using subscription-required Windows OS for Windows
graphics workstations, it now uses the same base OS as standard
Windows workstations and installs the required Nvidia graphics
drivers manually as part of the startup script.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
AWS: update README.md

Subscriptions are no longer required by Windows workstations and
the README file has been updated to reflect this.

Signed-off-by: Edwin-Pau <epau@teradici.com>

shermanyin commented on modules/aws/win-gfx/win-gfx-startup.ps1.tmpl:
```diff
@@ -60,6 +65,50 @@ function Retry([scriptblock]$Action, $Interval = 10, $Attempts = 30) {
+    cd $NVIDIA_DIR
+    & .\nvidia-smi.exe
+    return $?
+    return $false
```
why are there 2 returns here?

Edwin-Pau replied:
I reused the same function as GCP, should I make a separate commit to remove both?

shermanyin replied:
can you confirm what the return $_ gives you in both cases (before and after install)

The 2nd line should not be there. The GCP version needs to fix this too as it will never reach the 2nd line.

Edwin-Pau replied:
I'll correct this and force update this branch.

Tested by calling the function before and after:
```
Nvidia-is-Installed
Nvidia-Install
Nvidia-is-Installed
```

Result:
```
False
################################################################
Install NVIDIA GRID Driver
################################################################
<driver install log omitted here>

Nvidia-is-installed True
True

Commit:
AWS: update Windows base image (single and lb)

With the win_gfx base image change, both Windows workstations now use
the same Windows Server 2019 base image. They have been updated to the
latest, "Windows_Server-2019-English-Full-Base-2020.03.18". DC Windows
VM has also been updated to the latest for lb deployment.

Signed-off-by: Edwin-Pau <epau@teradici.com>
